src/server/api/schema.js

```diff
@@ -1080,7 +1080,7 @@ const rootMutations = {
           // run interaction step handler
           try {
             const handler = require(`../../integrations/action-handlers/${interactionStepAction}.js`);
-            handler.processAction(
+            await handlerprocessAction(
```

Comment:
Are you missing a file and an import?  handlerprocessAction isn't defined.

Reply:
SHA e54abc7e2ea7f163932038e8aad3131c028dab8e